docs/journalist/ending_session.rst

```diff
@@ -6,8 +6,6 @@ Ending your session
 When you are finished using SecureDrop Workstation, close the SecureDrop Client window and shut the computer down completely. This is to take advantage of the protections of full-disk encryption, and to avoid unauthorized access to the Workstation and the files and materials on it, which include any messages and submissions that you have downloaded.
 
 To shut down the computer, click your username in the top righthand corner of
-your screen, and select **Shut Down** from the context menu. Or, in the Qubes
-Applications menu, scroll to the bottom of the menu and select **Log Out > Shut
-Down**.
+your screen, and select **Shut Down** from the menu.
 
```

Comment:
(Technically this is not a [context menu](https://en.wikipedia.org/wiki/Context_menu) but just a regular menu.)